tutorial_hltau.ipynb

```diff
@@ -94,7 +94,7 @@
    "source": [
     "Sample data for the tutorial is the protoplanetary disk \"**HL Tau**\" observed at Band 6 with long baselines of ALMA 12m array as part of [science verification](https://almascience.nao.ac.jp/alma-data/science-verification). The calibration information can be obtained from [CASA Guide](https://casaguides.nrao.edu/index.php?title=ALMA2014_LBC_SVDATA#HL_Tau_-_Protoplanetary_Disk), and the data can be downloaded from the link below: \n",
     "\n",
-    "https://alma-dl.mtk.nao.ac.jp/ftp/alma/sciver/HLTauBand6/HLTau_Band6_CalibratedData.tgz\n",
+    "https://almascience.nao.ac.jp/almadata/sciver/HLTauBand6/HLTau_Band6_CalibratedData.tgz\n",
```

Comment:
Thank you for catching this! I also noticed broken URL recently and fixed it yesterday. This and the same URL below have already been fixed in main. I'm not sure if this causes conflict. If conflict happens, I might ask you to resolve the conflict by merging the latest main.

Reply:
It seems this change doesn't cause conflict.